CompanyNormalization.md

```diff
@@ -92,3 +92,5 @@ Lookup responses are identical to search responses in structure, with the `maste
 The response from the Company Normalization Service is versioned with the current version being 1.0. The CompanyDepot data is also versioned, and we will always use the latest.
 
 Our general versioning strategy is available [here](/Versioning.md).
+
+[1] This number is now deprecated and set to empty string. It would be removed from later version.
```

Comment:
I would be more explicit and mention duns_number. And I would say "the string will always return empty". minor grammar. I think it would read better with: "It *will* be removed from later versions."